src/find-specs.js

```diff
@@ -0,0 +1,74 @@
+const fetch = require("node-fetch");
+
+const specs = require("../index.json");
+
+const nonBrowserSpecWgs = ["Accessibility Guidelines Working Group", "Accessible Platform Architectures Working Group", "Automotive Working Group", "Dataset Exchange Working Group", "Decentralized Identifier Working Group", "Distributed Tracing Working Group", "Education and Outreach Working Group", "JSON-LD Working Group", "Publishing Working Group", "Verifiable Credentials Working Group", "Web of Things Working Group"];
+const watchedBrowserCgs = ["Web Platform Incubator Community Group", "Web Assembly Community Group", "Immersive Web Community Group", "Audio Community Group", "Privacy Community Group", "GPU for the Web Community Group"];
+
+const canonicalizeGhUrl = url => (url.indexOf("github.io") > 0 && url.split("/").length === 4 ? url + '/' : url).replace('http:', 'https:');
+const canonicalizeTRUrl = url => url.replace('http:', 'https:');
+
+Promise.all(
+  ["https://w3c.github.io/validate-repos/report.json",
+   "https://w3c.github.io/spec-dashboard/repo-map.json"].map(
+     dataUrl => fetch(dataUrl)
+       .then(r => r.json())
+   )
+).then(([{groups, repos}, specRepos]) => {
+  const wgs = Object.values(groups).filter(g => g.type === "working group" && !nonBrowserSpecWgs.includes(g.name));
+  const cgs = Object.values(groups).filter(g => g.type === "community group" && watchedBrowserCgs.includes(g.name));
+
+  // WGs
+  // * check repos with w3c.json/repo-type includes rec-track
+  const wgRepos = [].concat(...wgs.map(g => g.repos.map(r => r.fullName)))
```

Comment:
Why the leading `[].concat` here? It looks like you already have an array you're concatenating with, and if not `Array.from` is clearer.

Reply:
`[].concat(...` is what I've been using flatten array of arrays, but I see `Array.flat()` is now available, will switch to that